src/components/views/rooms/MessageComposerFormatBar.js

```diff
@@ -85,8 +85,8 @@ class FormatButton extends React.PureComponent {
 
         return (
             <AccessibleTooltipButton
-                as="span"
-                role="button"
+                element="button"
+                type="button"
```

Comment:
given that AccessibleTooltipButton's manage their own accessibility for aria/keyboard handling it may cause confusion to accessible technologies to both do that and use an actual `<button />`

Reply:
(arguably we should be ditching AccessibleButton and just styling real buttons to best support accessibility but the code dates from when styling buttons across browsers did not work consistently)

Reply:
As far as I know using `type="button"` should not interefere with accessible technologies. 

If that attribute is omitted and that `<button>` is used within a `<form>` it will be treated as a `<button type="submit">` by default which is not the behaviour we expect here.
It might be extraenous in that situation and I'd be happy to remove it. Let me know what'd be the best way forward